index.html

```diff
@@ -229,6 +207,27 @@ <h2>
               </ul>
           </div>
         </div>
+
+        <section class="contact-me">
+          <h3 class="contact-me-header">
+           I'm always interested in hearing about new projects,
+            so if you'd like to chat please get in touch.
+          </h3>
+          <form action="#" method="post" class="form-contact-me">
+            <input type="text"  name="username" placeholder="Full name" id="name" required>
```

Comment:
- [ ] Please make sure the name field only accepts up to 30 characters to fulfill the project requirements.
    - You can use this attribute `maxlength="30"`